revolv/static/home.scss

```diff
@@ -52,6 +62,46 @@ body {
 
 // --- Cover Image Styles ----------------------------------------------------------------
 
+.cover {
+    width: 100%; 
+    height: 600px;
+    overflow: hidden;
+    position: relative;
+
+    h1 {
+        color: white;
+        text-align: center;
+        text-shadow: 2px 2px black;
+    }
+
+    p {
+        font-size: $revolv-cover-inset-paragraph-size;
+        line-height: $revolv-cover-inset-paragraph-size + 6px;
+        text-align: center;
+        text-shadow: 2px 2px black;
+        font-weight: 200;
+        color: white;
+        padding-top: 30px;
+        line-height: 30px;
+        margin: 0;
+    }
+
+    .shade {
+        @include dark-shade(0.3);
+    }
+
+    .cover-inset {
+        @include absolute-center();
+        width: 600px;
+        z-index: 1;
+        text-align: center;
+    }
+
+    .cover-video {
+        width: 100%;
+    }
+}
+
 .cover-image {
```

Comment:
Cool cool. Also you could make a gist/pastebin/google doc!
On Sun, May 3, 2015 at 10:46 PM Andy Qin <notifications@github.com> wrote:

> In revolv/static/home.scss
> <https://github.com/calblueprint/revolv/pull/263#discussion_r29566978>:
>
> > +    .shade {
> > +        @include dark-shade(0.3);
> > +    }
> > +
> > +    .cover-inset {
> > +        @include absolute-center();
> > +        width: 600px;
> > +        z-index: 1;
> > +        text-align: center;
> > +    }
> > +
> > +    .cover-video {
> > +        width: 100%;
> > +    }
> > +}
> > +
> >  .cover-image {
>
> Yep, don't technically need any of the cover-image HTML or styles.
> However, would like to look at them for reference in moblifying. I think
> I'll just delete them so master is clean, and look at my older branches for
> reference
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/calblueprint/revolv/pull/263/files#r29566978>.
>